README.md

```diff
@@ -29,21 +29,28 @@ $ git clone https://github.com/AcompanhaLegis/docs.git
 $ cd docs
 </code>
 
-#### 3. Build the docker image from Dockerfile
+#### 3. You must activate the venv bin, it will be activated using
 
 <code>
-$ sudo docker build -t me-adota-docs .
+$ virtualenv venv
+```
```

Comment:
Okay, there's a bug here. I think there's no need of lines 36 and 37.

Reply:
fixed